src/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { useContext } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { BreadcrumbsContext } from '../../contexts/BreadcrumbsContext';
+import styles from './Breadcrumbs.module.css';
+
+const Breadcrumbs = () => {
+	const { breadcrumbs } = useContext(BreadcrumbsContext);
+	const navigate = useNavigate();
+
+	const onBreadcrumbClick = (url: string, index: number) => {
+		navigate(url);
+	}
+
+	return (
+		<nav>
+			<ul className={styles.list}>
+				{breadcrumbs.map(({ url, name }, index) => (
+					<li key={index}>
+						<button className={index === breadcrumbs.length - 1 ? styles.breadcrumbActive : ''} onClick={() => onBreadcrumbClick(url, index)}>{name}</button>
```

Comment:
For this libraries like "class names" can be used https://www.npmjs.com/package/classnames